README.md

```diff
@@ -75,35 +69,33 @@ You will be prompted to install the Python dependencies during the first use.
   - [Tutorial for formatting Adafruit CPX for CircuitPython](https://learn.adafruit.com/welcome-to-circuitpython/installing-circuitpython)
   - [Download Firmware .uf2 file](https://learn.adafruit.com/adafruit-circuit-playground-express/circuitpython-quickstart)
   - [Download the latest version of the Adafruit CPX library](https://learn.adafruit.com/welcome-to-circuitpython/circuitpython-libraries)
-- For developers:
-  - [Steps to run the extension locally](/docs/developers-setup.md)
 
-## How to use
+### How to use
 
 To use Device Simulator Express, install the extension from the marketplace and reload VS Code.
 
-### 1. Start with the "New File" Command.
+#### 1. Start with the "Device Simulator Express [Circuit Playground Express]: New File" Command.
 
-1. Type in `"Device Simulator Express: New File"` in the command palette (`CTRL+SHIFT+P` to open the command palette).  
+1. Type in `"Device Simulator Express: [Circuit Playground Express] New File"` in the command palette (`CTRL+SHIFT+P` to open the command palette).  
    <img alt='"New File" animation' src=https://raw.githubusercontent.com/microsoft/vscode-python-devicesimulator/dev/assets/readmeFiles/newFile.gif>
 2. Name and save your file somewhere, and we’re good to go! (3)
-3. Start with some examples: you can find examples files and tutorials inside the comments, as well as in the notification pop up when you run the `"Device Simulator Express: New File"` Command.
+3. Start with some examples: you can find examples files and tutorials inside the comments, as well as in the notification pop up when you run the `"Device Simulator Express: [Circuit Playground Express] New File"` Command.
 
 <img alt='How to find example code screenshot' src=https://raw.githubusercontent.com/microsoft/vscode-python-devicesimulator/dev/assets/readmeFiles/findExamples.jpg>
 
-### 2. Start from an existing python file.
+#### 2. Start from an existing python file.
 
 1. Open the folder or your .py file in Visual Studio Code.
-2. Run `open Simulator` from the command palette or icon in the editor toolbar.
+2. Run `Open Simulator` from the command palette or icon in the editor toolbar.

```

Comment:
It seems at other places in the README we use `Device Simulator Express: [Circuit Playground Express]` followed by the command, instead of only the command.